src/daemon/db_util.rs

```diff
@@ -142,6 +145,25 @@ async fn transcode_into_forest_car(from: &Path, to: &Path) -> anyhow::Result<()>
     Ok(())
 }
 
+pub fn cache_tipset_keys<DB>(
+    state_manager: Arc<StateManager<DB>>,
+    ts: &Tipset,
+) -> anyhow::Result<()>
+where
+    DB: fvm_ipld_blockstore::Blockstore,
+{
+    tracing::info!("Caching tipset keys for EthAPIs...");
```

Comment:
should this be an `info` log? that's normally for user-visible logs, like `println`

Reply:
Seems like a fine idea to have this as `info`, assuming this is also going to be configurable. Without the above cache one can't use eth api endpoints. It does not seem very noisy.